src/applications/disability-benefits/686/selectors/index.js

```diff
@@ -0,0 +1,16 @@
+// selectors for the `authorization686` state
+import { createStructuredSelector } from 'reselect';
+
+export const get686AuthorizationState = createStructuredSelector({
```

Comment:
I'm not sure how the memoization will work here since the full state is being passed into every input selector. Been a while since I've used `reselect` but I think breaking these deep queries into smaller selectors might be a better approach. e.g. `state.user` could be set up as 

`memoizedUserSelector = createStructuredSelector({user: state.user})` 

and reused in 

`selectUserStatus = createSelector(memoizedUserSelector, user => profile.status)`
`selectIsVerified = createSelector(memoizedUserSelector, user => profile.verified`

Not sure about this, though- My understanding is that it memoizes off of the arguments and the root `state` should fail an identity check on every `mapStateToProps` call.

you could easily test this by adding a console log to selector passed into reselect and just calling the selector a couple of times. first with the state and then with `{...state}`. nothing would change identity but the outer state object.

Reply:
Also recommend not spinning your wheels too much on this unless you're set on getting good at reselect- in case  it works and you want to move on 👍

Reply:
Before I can get good at reselect I first have to get adequate at it :) I'm gonna merge this and save your notes for further review.